app/services/linked_resource_builder.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+class LinkedResourceBuilder
```

Comment:
What's the purpose of this abstraction? Wouldn't it be less code to just `LinkedResourceFactory.new(resource: resource).to_json, rather than `LinkedResourceBuilder.new(resource: resource).build`?

Reply:
Certainly, yes.  I believe that my reasoning here was that the `LinkedResourceBuilder` remain structured along consistent lines with that of the `ManifestBuilder`.  Both model a service which transforms a Valkyrie resource into an RDF graph (rather than simple serialization to JSON), and, perhaps, there might be some interfaces which could be generalized in a future sprint.